src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala

```diff
@@ -566,6 +160,9 @@ object Initializer extends SparkSessionWrapper {
   private def initConfigState(debugFlag: Boolean,organizationID: Option[String],apiUrl: Option[String]): Config = {
     logger.log(Level.INFO, "Initializing Config")
     val config = new Config()
+    val orgId = if (config.isLocalTesting) System.getenv("ORGID") else {
+      getOrgId
+    }
```

Comment:
let's remove this reference as isLocalTesting is deprecated. Let's just continue to use the `getOrgId` function as we are below.